pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +361,73 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
+	args := []string{"dashboard", "get-rgw-api-access-key"}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	out, _ := cephCmd.Run()
+	if string(out) == "" {
+		return false
+	} else {
+		return true
+	}
+}
+
+func enableRGWDashboard(context *Context) error {
+	if checkDashboardUser(context) {
+		logger.Debug("dashboard already enabled for rgw")
+		return nil
+	}
+	user := ObjectUser{
+		UserID:      DashboardUser,
+		DisplayName: &DashboardUser,
+		SystemUser:  true,
+	}
+	u, errCode, err := CreateUser(context, user)
+	if err != nil || errCode != 0 {
+		return errors.Wrapf(err, "error creating %q user", DashboardUser)
+	}
+	args := []string{"dashboard", "set-rgw-api-access-key", *u.AccessKey}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	_, err = cephCmd.Run()
+	if err != nil {
+		return errors.Wrap(err, "error setting user accesskey")
```

Comment:
ditto

Reply:
For which user? Note in failure message.